src/Features/Core/Portable/xlf/FeaturesResources.de.xlf

```diff
@@ -642,7 +642,7 @@ Stellen Sie sicher, dass Sie den Bezeichner "tt" für Sprachen verwenden, für d
       </trans-unit>
       <trans-unit id="Modifying_source_file_will_prevent_the_debug_session_from_continuing_due_to_internal_error">
         <source>Modifying source file '{0}' will prevent the debug session from continuing due to internal error: {1}</source>
-        <target state="needs-review-translation">Durch das Ändern der Quelldatei "{0}" wird verhindert, dass die Debugsitzung aufgrund eines internen Fehlers fortgesetzt wird: {1}.</target>
+        <target state="needs-review-translation">Durch das Ändern der Quelldatei "{0}" wird verhindert, dass die Debugsitzung aufgrund eines internen Fehlers fortgesetzt wird: {1}</target>
```

Comment:
The test is green after this change if the msbuild property `EnableXlfLocalization` is set to true.

Reply:
I don't think XLF files should only be manually modified by a localization team member.

Reply:
The loc team needs to be in the loop for sure. If they want me to revert the change I can do so easily.

> A better approach IMO is to use the English version of resources in the tests.

Not in this case because the failing test revealed a wrong translation. The dot is superfluous because you end up with `!.` at the end of the sentence.

`ENC0080: Durch das Ändern der Quelldatei \"src.cs\" wird verhindert, dass die Debugsitzung aufgrund eines internen Fehlers fortgesetzt wird: System.NullReferenceException: NullRef!.`

Reply:
There was a discussion somewhere before on a similar issue and (if I recall correctly) the loc team said some languages require different punctuation or something of that kind.

Reply:
@JoeRobich My explanation of what is going on here:

The ` {1}` is replaced by a formatted exception in the form: `ExcpetionType: ExceptionMessage`. *ExceptionMessage* includes a sentence closing punctuation mark (at least in the BCL). The German translation adds a superfluous dot punctuation mark.